packages/mdc-chips/chip/component.ts

```diff
@@ -192,8 +192,8 @@ export class MDCChip extends MDCComponent<MDCChipFoundation> implements MDCRippl
         this.emit<MDCChipRemovalEventDetail>(
           strings.REMOVAL_EVENT, {chipId: this.id, root: this.root_}, true /* shouldBubble */);
       },
-      notifySelection: (selected) => this.emit<MDCChipSelectionEventDetail>(
-          strings.SELECTION_EVENT, {chipId: this.id, selected}, true /* shouldBubble */),
+      notifySelection: (selected, shouldHandle) => this.emit<MDCChipSelectionEventDetail>(
```

Comment:
Since you use the word `shouldIgnore` everywhere else, I think `shouldHandle` --> `shouldIgnore`

Reply:
ah you're right! thanks